lua/gatekeeper/lpmlib.lua

```diff
@@ -20,3 +20,29 @@ function lpm_add_tbl8s(ip_addr, prefix_len, prefixes)
 	return 1
 end
 
+function lpm6_add_tbl8s(ip6_addr, prefix_len, prefixes)
+	local prefix
+	local masked_ip
```

Comment:
These variables are only needed inside of the `while` loop, so move their declaration there.